### PR TITLE
Fixes meeting 01 03

### PR DESCRIPTION
**Pull request by shukon**
fix timeouts in performance-table
combine local search and random search (sorted) in configurator footprint
fix `min_runs` in meta-info by fixing all statistics there to the best run (adding comparison all/best/worst in next update)
fix tooltip for conf-footprint
fix threshold for lpi-entries

### CODE REVIEW

**Comment by shukon** [comment]
@AndreBiedenkapp oh, you did the same thing just now? I'll try to pick apart what's still new.

File: cave/analyzer.py [hunk 1]
```diff
@@ -420,7 +438,7 @@ def fanova(self, incumbent, marginal_threshold=0.05):
     def local_epm_plots(self):
         plots = OrderedDict([])
         self.parameter_importance("lpi", self.incumbent, self.output)
-        for p, i in [(k, v * 100) for k, v in sorted(self.pimp.evaluator.evaluated_parameter_importance.items(),
```
Comment (shukon) [inline cave/analyzer.py]: @AndreBiedenkapp why did you multiply with 100 here? It's only used in the for-loop as the `i`, which is not used.

**Comment by shukon** [comment]
I think this is still worth a review, since I addressed a few more points from the meeting.
We should treat all parameter importance-dicts the same way. So far I understood they are internally dealt with between 0 and 1 and displayed between 0 and 100 for convenience. Do you prefer to handle them internally between 0 and 100 too?

File: cave/analyzer.py (hunk 1 above)
Comment (AndreBiedenkapp) [reply cave/analyzer.py]: That was an error on my part, sorry.

**Pull request merged**